### ISSUE TITLE
Migrate opencats styling to Bootstrap

### ISSUE THREAD

**Issue by RussH** [opencats/OpenCATS#169]
@libregeek you had tried this previously - do you have a fork we can hack on to progress it?

**Comment by libregeek** [comment]
I need to check. I have lots of unmerged changes from upstream. Please give couple of days. I think have one with fontawesome also..

**Comment by libregeek** [comment]
It seems I lost all those changes when I switched my development machine last month. Now I'm stuck with #171 on the new desktop.

**Comment by RussH** [comment]
arrrrgh!



----
Russ

**Comment by Bloafer** [comment]
I could look into this if you want

**Comment by RussH** [comment]
Yes please, not sure of the best way to collaborate on a fork though...

**Comment by Bloafer** [comment]
Was just testing a few things, I'll try and make a branch from master and see where I get to

**Comment by Bloafer** [comment]
Once you are happy with the [PR](https://github.com/opencats/OpenCATS/pull/174) and it is merged into master I will start working on the main interface

**Comment by RussH** [comment]
Thanks @Bloafer , Matias will code-review your initial commit first..

**Comment by RussH** [comment]
hey @Bloafer - hesitate to ask but are you still taking a look at this - really really hope so!

**Comment by Bloafer** [comment]
I have actually been quite busy, start of the year everyone wants things done last year. I could start looking into this a bit more.

**Comment by Bloafer** [comment]
@RussH I have just commit some code on the `bootstrap-ui` branch, it's work in progress but will make templating much easier hopefully

**Comment by RussH** [comment]
Hi @Bloafer thanks for that - btw we're tending to have discussions in slack - if you want me to invite you to the team just drop me though your email address to russh@opencats.org. Sure sounds like you have allot on though!

**Comment by traderboy** [comment]
It looks like development has stopped on the bootstrap changes in the bootstrap-ui branch for over a year now.  The opencats2017 Theme looks great but is only applied to the Dashboard, Activities, and Settings tabs.  Will the other tabs be supported anytime soon?  Or has this effort completely ended?

**Comment by RussH** [comment]
Russell Hay
Sent from Mailspring, the best email app for work
E // russellh@ysmail.net (https://link.getmailspring.com/link/local-421c4075-2061-v1.1.4-22d9f20d@russh-ThinkPad-T61/0?redirect=mailto%3Arussellh%40ysmail.net&recipient=reply%2B0004507caa98efab3fbc140a42e0755282380649025f15a192cf0000000116c0d66292a169ce0b79f8f7%40reply.github.com)
P // +447967275932 (tel:+447967275932)

**Comment by RussH** [comment]
Hi @traderboy after the good work from @Bloafer it was kinda put on hold as there was a wholesale refactoring and move into Symfony framework planned.. which would replicate or replace the bootstrap coding (this work purely by @mlespiau ). Due to some personal commitments, this hasn't progressed to completion yet.

**Comment by ftntming** [comment]
And we're now in year end 2020. This feature is still not done

**Comment by shantadam** [comment]
Your concern is as equal as the lack of voluntary project contribution hands, Alan.

The good news is, Alan, OpenCATS member group have recently managed to contribute into the upgrade of its much awaited PHP 7.2 component support and many other bug fixes. We’re thankful for their efforts.

A non-funded project support by voluntary members can certainly expect extensive delays in delivering new features. Don’t you agree, Alan?

Forgive the miss Alan, but I do have questions for you:

  *   How is OpenCATS facilitating you or your organization so far in your daily activities?
  *   Is the need for a mobile version of OpenCATS?
  *   Can you or any capable volunteers contribute to the project on their spare time?

Regards,
--
Shant Adam,
T: (778) 877-2028
E: shant.adam@mxcgi.com<mailto:shant.adam@mxcgi.com>
--
DISCLAIMER - This message contains confidential information and is intended only for the individual named. If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. E-mail transmission cannot be guaranteed to be secure or error-free as information could be intercepted, corrupted, lost, destroyed, arrive late or incomplete, or contain viruses. The sender therefore does not accept liability for any errors or omissions in the contents of this message, which arise as a result of e-mail transmission.

**Comment by anonymoususer72041** [comment]
As discussed via email with @RussH, we will migrate to TailwindCSS instead.
